Avoid dependency on system/library/c++-runtime (Studio)
It's unknow now why it's being added.

--- firefox-91.0.1/toolkit/library/moz.build.~1~	2021-09-02 15:32:06.152235516 +0000
+++ firefox-91.0.1/toolkit/library/moz.build	2021-09-02 15:32:49.229619527 +0000
@@ -302,6 +302,15 @@
 if CONFIG["MOZ_ENABLE_LIBPROXY"]:
     OS_LIBS += CONFIG["MOZ_LIBPROXY_LIBS"]
 
+if CONFIG['OS_ARCH'] == 'SunOS':
+    OS_LIBS += [
+        'elf',
+    ]
+    if CONFIG['CC_TYPE'] in ('clang', 'gcc'):
+        OS_LIBS += [
+            'demangle',
+        ]
+
 if CONFIG["OS_ARCH"] == "FreeBSD":
     OS_LIBS += [
         "util",
